src/Rotations.jl

```diff
@@ -48,6 +53,8 @@ export
         valid_rotation,
 
         # extra quaternion operations
-        rot_angle  # N.B. angle(Quaternions) is different and defined in Quaternions
+        rot_angle,  # N.B. now depricated in favour of angle
```

Comment:
I've un-depricated this to export rot_angle and rot_axis rather than angle
and axis as discussed..

On 3 June 2016 at 13:13, Chris Foster <notifications@github.com> wrote:

> In src/Rotations.jl
> <https://github.com/FugroRoames/Rotations.jl/pull/3#discussion_r65651886>:
>
> > @@ -48,6 +53,8 @@ export
> >          valid_rotation,
> >
> >          # extra quaternion operations
> > -        rot_angle  # N.B. angle(Quaternions) is different and defined in Quaternions
> > +        rot_angle,  # N.B. now depricated in favour of angle
>
> typo "deprecated"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/3/files/da78deba58bd4f502ad6fcc11453606a47d9a354#r65651886>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQAxNNOBs3PKjlkswd-QPew8hVbmSU-fks5qH5v3gaJpZM4ItIfo>
> .
>